install.py

```diff
@@ -17,15 +17,16 @@
 
 DOWNLOAD_URL_FORMAT = 'https://github.com/grow/grow/releases/download/{version}/{name}'
 RELEASES_API = 'https://api.github.com/repos/grow/grow/releases'
-ALIAS_FILES_LINUX = ['.bash_aliases', '.bash_profile', '.profile', '.bashrc']
-ALIAS_FILES_MAC = ['.bash_profile']
-ALIAS_FILES_DEFAULT = '.bashrc'
-ALIAS_RE = re.compile(r'^alias grow\=([\'"])(.*)\1$', re.MULTILINE)
+RC_FILES_LINUX = ['.bash_profile', '.profile', '.bashrc']
+RC_FILES_MAC = ['.bash_profile']
+RC_FILE_DEFAULT = '.bashrc'
 
 if 'Linux' in platform.system():
     PLATFORM = 'linux'
+    BIN_PATH = '/usr/local/bin/grow'
```

Comment:
The issue with using /usr/local/bin/ is that it requires `sudo` to write to that directory. I'd recommend keeping the old `$HOME/bin` location and having the user include that in their $PATH.

Reply:
I think having a single known location would be easier to maintain, and I think I generally like `$HOME/bin` for installers that are automatically installed via `curl ... | bash`, since it's less scary.

Reply:
I've updated it to only install to the `~/bin` by default on all OSes and does the path append for those that don't have that directory in the `PATH` already.